packages/babel-plugin-extract-messages/test/fixtures/js-with-macros.js

```diff
@@ -1,4 +1,4 @@
-import { t, defineMessage, msg, useLingui } from "@lingui/macro"
```

Comment:
do we need plural here?

Reply:
Yes, it was used in the file but was not referenced from import. New implementation now catches this